### PR TITLE
KF6 WIP

### PR DESCRIPTION
**Pull request by a17r**

### CODE REVIEW

**Review by thesamesam — APPROVED** [review]

File: kde-frameworks/extra-cmake-modules/extra-cmake-modules-9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+PYTHON_COMPAT=( python3_{10..12} )
+QTMIN=6.6.0
+inherit cmake frameworks.kde.org python-any-r1
+
+DESCRIPTION="Extra modules and scripts for CMake"
+HOMEPAGE="https://invent.kde.org/frameworks/extra-cmake-modules"
+
+LICENSE="BSD"
+SLOT="5" # TODO: move to SLOT=0 and set it in eclass
```
[the remark this answers is not in the document]
Comment (a17r) [reply kde-frameworks/extra-cmake-modules/extra-cmake-modules-9999.ebuild:14]: just as long as this is a wip.

File: kde-frameworks/kwallet/metadata.xml [hunk 2]
```diff
@@ -10,6 +10,7 @@
 	</upstream>
 	<use>
 		<flag name="gpg">Support wallets with GnuPG encryption in addition to the default blowfish-encrypted file</flag>
+		<flag name="kf6compat">Disable components colliding with KF6, depend on KF6 components instead</flag>
```
Comment (thesamesam) [inline kde-frameworks/kwallet/metadata.xml:13]: I don't love the name but I don't have a better suggestion.

File: app-crypt/qgpgme/qgpgme-1.23.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+# Bump FUDGE if a release is made which breaks ABI without changing SONAME.
+# (Reset to 0 if FUDGE != 0 if libgpgme/libgpgmepp/libqpggme change.)
+SLOT="1/11.6.15.2"
+KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~loong ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+# - On each bump, update dep bounds on each version from configure.ac!
+RDEPEND="
+	~app-crypt/gpgme-${PV}:${SLOT}
```
[the remark this answers is not in the document]
Comment (a17r) [reply app-crypt/qgpgme/qgpgme-1.23.1.ebuild:42]: I don't think it does that at all right now ;)

File: kde-frameworks/extra-cmake-modules/extra-cmake-modules-9999.ebuild (hunk 1 above)
[the remark this answers is not in the document]
Comment (a17r) [reply kde-frameworks/extra-cmake-modules/extra-cmake-modules-9999.ebuild]: https://github.com/gentoo/gentoo/pull/33989

File: kde-frameworks/kwallet/metadata.xml (hunk 2 above)
Comment (a17r) [reply kde-frameworks/kwallet/metadata.xml:13]: `kf6allowance`?

**Pull request merged**